testData/testConfig.json

```diff
@@ -2,6 +2,12 @@
   "user-namespace": "jupyter-pods",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
+  "gen3-license-user-maps-dynamodb-table": "gen3-license-user-maps",
+  "gen3-license-type": "TEST-LICENSE",
+  "gen3-g3auto-name": "test-license-g3auto",
+  "gen3-max-license-ids": 6,
+  "gen3-g3auto-key": "test_license.txt",
+  "gen3-license-file-path": "stata.lic",
```

Comment:
can we nest of all these configs under a `license` block? similar to the existing "sidecar" and "nextflow" config blocks https://github.com/uc-cdis/hatchery/blob/master/doc/howto/configuration.md

Reply:
Yes, good idea! Most of the license config has moved to the container sections. This will allow configuration of multiple licenses for a given commons. 

The dynamodb table name and global secondary index are still in the main hatchery config section - only one table is needed for multiple licenses or multiple commons.